### ISSUE TITLE
os9_command failing with error about dictionary (same as https://github.com/ansible-collections/dellemc.os10/issues/43)

### ISSUE THREAD

**Issue by igloo777** [ansible-collections/dellemc.os9#15]
##### ISSUE TYPE
- Bug Report

##### COMPONENT NAME
os9_command

##### ANSIBLE VERSION
```
ansible 2.10.1

collection dellemc.os9       1.0.3
```

##### OS / ENVIRONMENT
Ubuntu 20.04

##### STEPS TO REPRODUCE
```
      - name: "Check all about SNMP"
        dellemc.os9.os9_command:
          commands: show running-config acl | grep SNMP
```

##### ACTUAL RESULTS
```
The full traceback is:
  File "/tmp/ansible_dellemc.os9.os9_command_payload_r44ruytp/ansible_dellemc.os9.os9_command_payload.zip/ansible/module_utils/basic.py", line 1879, in _handle_elements
    validated_params.append(type_checker(value, **kwargs))
  File "/tmp/ansible_dellemc.os9.os9_command_payload_r44ruytp/ansible_dellemc.os9.os9_command_payload.zip/ansible/module_utils/basic.py", line 1757, in _check_type_dict
    return check_type_dict(value)
  File "/tmp/ansible_dellemc.os9.os9_command_payload_r44ruytp/ansible_dellemc.os9.os9_command_payload.zip/ansible/module_utils/common/validation.py", line 431, in check_type_dict
    raise TypeError("dictionary requested, could not parse JSON or key=value")
fatal: [switch]: FAILED! => {
    "ansible_facts": {
        "discovered_interpreter_python": "/usr/bin/python3"
    },
    "changed": false,
    "invocation": {
        "module_args": {
            "commands": [
                "show running-config acl | grep SNMP"
            ],
            "interval": 1,
            "match": "all",
            "retries": 10
        }
    },
    "msg": "Elements value for option commands is of type <class 'str'> and we were unable to convert to dict: dictionary requested, could not parse JSON or key=value"
}

```

Actually issue code is in https://github.com/ansible-collections/dellemc.os9/blob/master/plugins/modules/os9_command.py
```
def main():
    """main entry point for module execution
    """
    argument_spec = dict(
        # { command: <str>, prompt: <str>, response: <str> }
        commands=dict(type='list', elements='dict', required=True),
```

**Comment by equinoxefr** [comment]
Hi @igloo777 

Same issue here. They changed the format of the command without update in documentation ;)

This worked for me, hope it can help.
```
 - name: check version
    dellemc.os9.os9_command:
      commands:
        - command: show version
```

**Comment by komalupatil** [comment]
Fix is available in collection v1.0.4.

**Comment by skg-net** [comment]
https://github.com/ansible-collections/dellemc.os9/releases/tag/v1.0.4
